ballerina/Ballerina.toml

```diff
@@ -37,5 +37,5 @@ path = "./lib/wss4j-ws-security-common-3.0.1.jar"
 [[platform.java17.dependency]]
 groupId = "org.apache.santuario"
```

Comment:
Dont we have a Ballerina.toml in build-config folder?

Reply:
There, it takes the dependency version from the gradle.properties file

Reply:
Ack. Forgot that it has only placeholders